ui-grid/ui-grid.d.ts

```diff
@@ -490,6 +490,10 @@ declare module uiGrid {
          */
         footerHeight?: number;
         /**
+         * returns or sets grid height in pixels
+         */
+        gridHeight?: number;
```

Comment:
@MatejQ I don't see an issue in including it, but the documentation should note that it is unsupported and may mess things up